### DISCUSSION TITLE
Add reviews and photos of POIs

### DISCUSSION THREAD

**Discussion by FinixFighter** [organicmaps/organicmaps#371] [Ideas]
As stated in the Telegram group, OMaps could use http://openplacereviews.org/ for both reviewing and adding photos to places mapped on OSM. This would be an amazing and very useful feature.

**Comment by richlv** [comment]
There's also https://mangrove.reviews/ , which at least seems to be promising more openness/privacy.
Don't know more about them, though.

**Comment by FinixFighter** [comment]
> There's also https://mangrove.reviews/ , which at least seems to be promising more openness/privacy.
> Don't know more about them, though.

It would be amazing to view reviews from more than one source. For example, there are apps to search for movies, and for every movie they show both IMDB, TMDB votes and reviews from multiple websites. In my opinion the tough part is to link an OSM point of interest with the relative object on a certain review website.
